### PR TITLE
qase-python-commons: fixed problem with rewrite fields

### PR DESCRIPTION
**Pull request by gibiw**
qase-python-commons: fixed problem with rewrite fields
--
If a test has qase ID then we skip fields like title, description, etc. 
If a test doesn't have qase ID then we assign fields like title, description, etc.

### CODE REVIEW

File: qase-python-commons/src/qaseio/commons/testops.py [hunk 1]
```diff
@@ -335,24 +345,32 @@ def _send_result(self, result: Result):
...
+                stacktrace=result.execution.stacktrace,
+                time_ms=result.execution.duration,
+                comment=result.message,
+                attachments=[attach.hash for attach in attached],
+                defect=self.defect,
+                steps=steps,
+                param=result.params
+            )
+
+            test_ops_id = result.get_testops_id()
```
Comment (NickVolynkin) [inline qase-python-commons/src/qaseio/commons/testops.py:359]: It's called `testops_id` in many places throughout the code.

File: qase-python-commons/src/qaseio/commons/testops.py [hunk 2]
```diff
@@ -335,24 +345,32 @@ def _send_result(self, result: Result):
             if result.get_suite_title():
                 case_data["suite_title"] = "\t".join(result.get_suite_title().split("."))
 
+            result_model = ResultCreate(
```
Comment (NickVolynkin) [inline qase-python-commons/src/qaseio/commons/testops.py:348]: At line 141 `result_model` is a `Map` object, and here it's `ResultCreate`.

File: qase-python-commons/src/qaseio/commons/testops.py (hunk 1 above)
Comment (gibiw) [reply qase-python-commons/src/qaseio/commons/testops.py]: fixed

File: qase-python-commons/src/qaseio/commons/testops.py (hunk 2 above)
Comment (gibiw) [reply qase-python-commons/src/qaseio/commons/testops.py:348]: We have fixed it in the new version.

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**